app/static/pages/volunteer/tabs_template/lesson_creation_tab.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<div layout="row" class="oppia-foundation-white-background">
+  <md-content class="oppia-foundation-horizontal-padding
+                     oppia-foundation-white-background
+                     volunteer-page-lesson-creation-tab-content"
+              layout="column"
+              layout-align="start">
+    <p class="oppia-foundation-no-margin-top">
+      Oppia’s lesson creation team is responsible for creating a set of highly
+      effective and enjoyable lessons that students around the world can use for
+      free. The team follows a tested process to generate effective lessons that
+      appeal to students from all backgrounds.
+    </p>
+    <p>
+      Our goal is to not only teach
+      students in an easily digestible way, but also encourage a positive
+      attitude towards learning. Currently, we're focusing on basic mathematics,
+      but intend to expand to other subjects in the future.
+    </p>
+    <ng-include src="volunteerWorkDescriptionUrls.lessonCreation"></ng-include>
+    <md-button class="oppia-foundation-green-button"
+               href="https://goo.gl/forms/otv30JV3Ihv0dT3C3">
```

Comment:
Use this form instead: https://goo.gl/forms/w1JtEtRDpGMCk3aB3

Reply:
Thanks for reminding me about this :D